src/main/sensors/gyro.h

```diff
@@ -58,6 +58,11 @@ typedef enum {
     GYRO_OVERFLOW_CHECK_ALL_AXES
 } gyroOverflowCheck_e;
 
+typedef enum {
+    BEFORE_STATIC_FILTERS = 0,
+    AFTER_STATIC_FILTERS
+} dyn_filter_location_e;
```

Comment:
`dyn_filter_location_e` isn't used anywhere. Removing the name and making it an anonymous enum will remove the ambiguity about it being used anywhere.

Reply:
If I simply delete the name, so it looks like:
```
typedef enum {
    BEFORE_STATIC_FILTERS = 0,
    AFTER_STATIC_FILTERS
} ;
```
When I compile I then get a mass of errors of this kind:
```
In file included from ./src/main/drivers/accgyro/accgyro.h:30:0,
                 from ./src/main/telemetry/ltm.c:50:
./src/main/sensors/gyro.h:64:1: warning: useless storage class specifier in empty declaration
```
Let me know how best to proceed.

Reply:
Done, gotcha, sorry I'm so slow